src/parser/parser-utils.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+const ParserUtils = {
+  BufferToStr: (buffer: ArrayBuffer): string => {
+    const textDecoder = new TextDecoder();
+    return textDecoder.decode(buffer);
+  },
+  StrToBuffer: (str: string): Uint8Array => {
+    const textEncoder = new TextEncoder();
+    return textEncoder.encode(str);
+  }
```

Comment:
TextDecoder and TextDecoder are not supported in IE11.
Either we say we don't support it on IE11 and add protections by just checking if TextDecoder and TextEncoder are in window.
Another option is to add a polyfill: https://gist.github.com/Yaffle/5458286

Reply:
fixed